File: programming-languages/Java/fullexample.java
import java.util.Scanner;

public class fullexample{
    public static int somar(int x, int y) {
        return x + y;
    }

    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);

        System.out.print("Digite o primeiro número: ");
        int a = scanner.nextInt();

        System.out.print("Digite o segundo número: ");
        int b = scanner.nextInt();

        int resultado = somar(a, b);

        if (resultado > 10) {
            System.out.println("A soma é maior que 10.");
        } else {
            System.out.println("A soma é 10 ou menor.");
        }

        for (int i = 0; i < 3; i++) {
            System.out.println("Contagem: " + i);
        }
    }
}

File: programming-languages/Python/full-example.py
def somar(x, y):
    return x + y

a = int(input("Digite o primeiro número: "))
b = int(input("Digite o segundo número: "))

resultado = somar(a, b)

if resultado > 10:
    print("A soma é maior que 10.")
else:
    print("A soma é 10 ou menor.")

for i in range(3):
    print("Contagem:", i)
